w3c.json

```diff
@@ -0,0 +1,6 @@
+ {
+    "group":      109735
+,   "contacts":   ["dontcallmedom"]
+,   "shortName":  "hit-test"
```

Comment:
as in https://github.com/immersive-web/dom-overlays/pull/31#pullrequestreview-413524734, I suggest removing `shortName` and adding yourself to the list of contacts

Reply:
I think the shortname is webxr-hit-test, see index.bs